llvm/include/llvm/ObjCopy/CommonConfig.h

```diff
@@ -191,6 +152,61 @@ struct NewSectionInfo {
   std::shared_ptr<MemoryBuffer> SectionData;
 };
 
+// Flags set by --set-section-flags or --rename-section. Interpretation of these
```

Comment:
The highlighted change is not really what I moved. It's probably the minimal diff. I needed NameMatcher definition to be before SectionPatternAddressUpdate and tried to do it in a sensible way so that similar definitions stay together.

Reply:
Would it be okay to just do the minimal thing possible and then if you still would like to reorder things, reorder them in a follow-up PR? That way, it's easier to see what's actually changed.

Reply:
All these changes are essential. NameMatcher needs NameOrPattern defined before it which in turn requires MatchStyle